### ISSUE TITLE
Output window no longer opens upon starting command

### ISSUE THREAD

**Issue by Jazarro** [morassman/process-palette#91]
I just upgraded from `0.17.1` to `0.18.1`. Before, when I ran a command it would automatically open the output window at the bottom of my screen, allowing me to observe the output of the process I just started. It now no longer opens that window. 

This is the command specification I'm using:

```JSON
{
    "namespace": "MY_NAMESPACE",
    "action": "pack-and-format-all",
    "command": "java -jar JARNAME.jar",
    "cwd": "{configDirAbsPath}/modules/",
    "keystroke": "ctrl-1",
    "menus": [
        "MY_NAMESPACE"
    ],
    "stream": true,
    "outputTarget": "panel",
    "autoHideOutput": true,
    "singular": true,
    "saveOption": "all"
},
```

I am using `Windows 10` and `Atom 1.55.0 x64`

**Comment by morassman** [comment]
Hi.

Thanks for reporting this. It should be fixed now in `0.18.2`.

**Comment by Jazarro** [comment]
Yes, it is fixed! Thank you very much for fixing this so quickly! Process Palette is a great package, stellar work!
